app/sidekiq/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -71,6 +71,13 @@ class SubmitForm0781 < Job
           bgjob_errors: bgjob_errors.merge(new_error)
         )
 
+        if Flipper.enabled?(:disability_compensation_use_api_provider_for_0781_uploads)
+          submission = Form526Submission.find(form526_submission_id)
+
+          provider = api_upload_provider(submission, FORM_ID_0781)
```

Comment:
Bahhh that's right we don't know which form actually failed here (0781 or 0781a) but since we're logging the same metric regardless and logging the actual error below it doesn't matter, so I think this is totally fine

Reply:
Yeah, that was my thinking as well. If we need to dig into the error more, we can find the form type on the submission.